assets/floormaps/hall/HallFloor9.js

```diff
@@ -4,32 +4,20 @@ import Svg, { G, Path, Text } from "react-native-svg";
 import { IndoorScenario } from "../../../components/IndoorDirections/IndoorScenario";
 import { Hall9Coordinates } from "../../../constants/Hall9Coordinates";
 
-export function HallFloor9() {
+export function HallFloor9(props) {
     const [floorNumber, setFloorNumber] = React.useState("");
-    const [to, setTo] = React.useState("");
-    const [from, setFrom] = React.useState("");
+    const [to, setTo] = React.useState(props.from);
+    const [from, setFrom] = React.useState(props.to);
     const rooms = Hall9Coordinates();
```

Comment:
<img width="750" alt="Screen Shot 2020-04-05 at 4 57 32 PM" src="https://user-images.githubusercontent.com/35850933/78509773-8c23a980-775e-11ea-9007-50a8adea2764.png">

Reply:
@mohanadarafe

Reply:
For this one, I dont believe it was directly related to this PR, this variable was already there before you guys added this feature, ignore it if you dont have time and I'll find a fix later. It showed up with the analysis of this PR because of the other issues introduced by the PR in this file.